### ISSUE TITLE
initialManifests not working as expected

### ISSUE THREAD

**Issue by wibed** [siderolabs/talos#7993]
i get mapping errors trying to install manifests on bootstrap

patch file:
```yaml
```

```yaml
```

**Comment by DmitriyMV** [comment]
@wibed 

Hey! Thanks for the report. I believe the culprit is here:
```yaml
...
      contents: |

        --
        apiVersion: v1
...
```

Removing this newline solves the issue but we probably need to trim those too on Talos end.

**Comment by wibed** [comment]
> @wibed If you don't mind - how did you generate this patch? Manually? The reason I'm asking is that I'm curious we we are inserting this newlines ourselves somewhere.

yes i did a "create-patch-command" (might be a cat idr) >> patch.yaml
